src/components/dialog/AttachGroupToUserDialog.vue

```diff
@@ -84,22 +84,23 @@ async function onSubmit() {
 
   const groupIdList = selected.value.map(({ objectId }) => objectId);
 
-  await UserService.attachGroups(userId.value, groupIdList)
-    .then(() => {
-      Notify.create({
-        type: 'positive',
-        message: t('AttachGroupToUserDialog.text.notifySuccess'),
-        html: true,
-      });
-      show.value = false;
-    })
-    .catch(() => {
-      Notify.create({
-        type: 'negative',
-        message: t('AttachGroupToUserDialog.text.notifyError'),
-        html: true,
-      });
-    });
+  await Promise.allSettled(groupIdList
+    .map((groupId) => UserService.addUserToGroup(userId.value, groupId)
```

Comment:
well you misplaced closed `)`, because on each user attachment you will close the dialog.
I prefere to close the dialog when all users is attached!

Reply:
But you can keep the notify error on each user attachements.

Reply:
It was not an error ;) I moved the closing action of the dialog